lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -2105,7 +2105,7 @@ public function newInstance(ClassMetadata $class)
      * INTERNAL:
      * Creates an entity. Used for reconstitution of persistent entities.
      *
-     * Internal note: Highly performance-sensitive method.
+     * {@internal Highly performance-sensitive method. }}
```

Comment:
Did something go wrong with closing brackets?

Reply:
From the phpdoc reference `To increase readability and ease parsing should the tag be terminated with a double closing brace, instead of a single one.`

Reply:
👍

Reply:
This really looks like a typo/bug, I would prefer not doubling it if it is not required (I see "should" above)...

Reply:
True, it does not look obvious from the first look. Here's a discussion about it https://github.com/phpDocumentor/fig-standards/pull/80. Until it's not changed, I would stick to the current syntax, tbh.